src/parse/asp/errors.go

```diff
@@ -44,6 +45,7 @@ func fail(pos Position, message string, args ...interface{}) {
 
 // AddStackFrame adds a new stack frame to the given errorStack, or wraps an existing error if not.
 func AddStackFrame(pos Position, err interface{}) error {
+	log.Debugf("error parsing build file: %v \n%v", err, string(debug.Stack()))
```

Comment:
this will log a bunch of times (once per asp scope), maybe we should log this when the assert fails or whatever